src/Microsoft.Azure.WebJobs.Host.Storage/WebJobs.Host.Storage.csproj

```diff
@@ -1,6 +1,4 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <Import Project="..\Microsoft.Azure.WebJobs.Shared\WebJobs.Shared.projitems" Label="Shared" />
```

Comment:
The Sanitizer was getting compiled into both WebJobs.Host and WebJobs.Host.Storage, and that created a compilation issue because the latter also has internals visible to the former. This shared reference isn't needed for now, so just removing. Could have also resolved by aliasing the assembly reference.

Reply:
Haven't looked, but were there any public types in the shared project? Just want to make sure we don't end up with a breaking change as a result.